index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for koa-verto 0.0
+
+import * as Koa from 'koa';
+
+export = setVersionHeaders;
+
+declare function setVersionHeaders(options?: verto.Options): Koa.Middleware;
```

Comment:
I've never worked with TS, so for me, this is bit unclear, but I don't understand why this function is here, as there is no such function in the original codebase (master branch).

Reply:
OK -- commit message changed with feat:  to the keyword..

Cheers